ext/nnstreamer/tensor_decoder/tensordec-directvideo.c

```diff
@@ -40,11 +41,59 @@ void fini_dv (void) __attribute__ ((destructor));
     GST_VIDEO_CAPS_MAKE ("{ RGB, BGRx, GRAY8 }") \
```

Comment:
when adding additional video format, you should update this macro.

Reply:
it's added